hack/templates/resources/90-shoot.yaml.tpl

```
@@ -88,19 +88,55 @@ spec:
         autoScalerMax: 2
         maxSurge: 1
         maxUnavailable: 0
-      # machineImage:
-      #   name: coreos
-      #   version: 2023.5.0
-      # labels:
-      #   key: value
-      # annotations:
-      #   key: value
-      # taints: # See also https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
-      # - key: foo
-      #   value: bar
-      #   effect: NoSchedule
-      % endif
-      zones: ${value("spec.cloud.aws.zones", ["eu-west-1a"])}
+       # kubelet:<% kubelet=value("spec.cloud.aws.workers.kubelet", {}) %>
```

```suggestion
      # kubelet:<% kubelet=value("spec.cloud.aws.workers.kubelet", {}) %>
```

Properly indent (repeat the same for all providers)